packages/react-native/React/Runtime/RCTJscInstanceFactory.h

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
shouldn't this be in the external pod? Or are we keeping it here while we support the old JSC and the plan is to move it away or delete it when we only have hermes or the external jsc pod from community?

Reply:
My idea was to keep it here until we fully extract it. The API is now relying on passing a C pointer: 

https://github.com/facebook/react-native/blob/af32fe878efd908d1f687bc23c5ae2529b7e32b5/packages/react-native/Libraries/AppDelegate/RCTRootViewFactory.mm#L267-L269

Once we fully remove JSC we can then leave only the hermes code